webapp/channels/src/components/integrations/abstract_oauth_app.tsx

```diff
@@ -167,58 +187,87 @@ export default class AbstractOAuthApp extends React.PureComponent {
         }
 
         const app = {
+            id: this.props.initialApp?.id || '',
             name: this.state.name,
             callback_urls: callbackUrls,
             homepage: this.state.homepage,
             description: this.state.description,
             is_trusted: this.state.is_trusted,
             icon_url: this.state.icon_url,
+            create_at: this.props.initialApp?.create_at || 0,
+            update_at: this.props.initialApp?.update_at || 0,
+            creator_id: this.props.initialApp?.creator_id || '',
+            client_secret: this.props.initialApp?.client_secret || '',
```

Comment:
I am not sure about passing these new values because previously we were not passing them. I would ask for a double check from internal mattermost member cc: @M-ZubairAhmed

Reply:
Please add partial to the parameter type so we can remove these defaults